jscomp/runtime/js_int.ml

```diff
@@ -116,32 +78,16 @@ before the decimal point.
 @raise RangeError if digits is not in the range accepted by this function (what do you mean "vague"?)
 
 {[
-  (* prints "1.2e+8" *)
```

Comment:
I can't comment on them, but there are some refs to `toPrecisionWithPrecision` that are now obsolete:

https://github.com/melange-re/melange/blob/564b9684891f4d32584aa957379f939e5d6edf2f/jscomp/runtime/js_int.ml#L72-L76